hack/tools/install.sh

```diff
@@ -124,6 +126,14 @@ function install_golangci-lint() {
   install_tar_binary "${INSTALL_PATH}" "https://github.com/golangci/golangci-lint/releases/download/${GOLANGCI_LINT_VERSION}/golangci-lint-${GOLANGCI_LINT_VERSION:1}-${OS}-${ARCH}.tar.gz" "golangci-lint-${GOLANGCI_LINT_VERSION:1}-${OS}-${ARCH}/golangci-lint"
 }
 
+function install_gomplate() {
+  INSTALL_PATH="${1}"
+
+  # gomplate includes library from no longer existing domain inet.af, and thus cannot be installed via go install
```

Comment:
Nit: Can we change comment to `gomplate must be installed via Github releases`

Reply:
I'm not sure I understand the request? You want me to drop the context of why it can't be installed?

Reply:
Yes because end maintainer doesn't need that deep of context of `inet.af` not existing. But feel free to leave in if you think it's worth it.